pom.xml

```diff
@@ -24,7 +24,7 @@
     <properties>
       <jenkins.version>1.580.1</jenkins.version>
       <java.level>6</java.level>
-      <workflow.version>1.1</workflow.version>
+      <workflow.version>1.4.2</workflow.version>
```

Comment:
`build` did not return an appropriate value in the ancient 1.1 release. 1.4.2 is the newest compatible with 1.580.x.